servers/rendering/renderer_rd/renderer_scene_render_rd.h

```diff
@@ -92,13 +244,49 @@ class RendererSceneRenderRD : public RendererSceneRender {
 	double time = 0.0;
 	double time_step = 0.0;
 
+	/* ENVIRONMENT */
+
+	RS::EnvironmentSSAOQuality ssao_quality = RS::ENV_SSAO_QUALITY_MEDIUM;
```

Comment:
I moved these here so they are accessible by our subclasses as I moved methods that only belong in our clustered forward renderer into that class. But this really needs to change to getters.
I also believe these can be moved into our effects class and in doing so simplifying the API of our effects class.

Reply:
Yeah it looks to me that these should be moved to their own class.

Reply:
I need to look into this further because some of these are a little entangled. I'd like to do these in future PRs.

Reply:
Managed to move all the SSEffects related settings into the SSEffects class. This is in a separate commit as it really stands apart from the main body of work.

There is more to move but as I think those deserve smaller PRs. This was the bigger one.